CorsixTH/Lua/app.lua

```diff
@@ -313,6 +313,39 @@ function App:init()
   return true
 end
 
+---
+-- Initialises a Lua DBGp client and
+-- connects it to a DBGp server so that
+-- it can debug lua scripts at runtime.
+-- @return error_message
```

Comment:
@Lego3 I was thinking of LuaDoc conventions when I chose this type of function comment, from now on I will use the LDocGen conventions. With the LDT plugin installed Eclipse can recognize any Lua comment type.

Eclipse does have a good DoxyGen plugin with support for Graphviz comment diagrams:
http://mcuoneclipse.com/2012/06/25/5-best-eclipse-plugins-1-eclox-with-doxygen-graphviz-and-mscgen/